src/pages/components/Components.tsx

```diff
@@ -191,6 +213,14 @@ const Components = (): JSX.Element => {
           navSecondaryItems={navItems}
           navTertiaryItems={headings}
         >
+          {/* Storybook Link */}
+          {(storybookComponent !== '' && (
+            <div className="docs-storybook-link">
+              <Link href={toStorybookLink(`${storybookComponent}${componentData?.name}--docs`)}>View in Storybook</Link>
+            </div>
+          )) ||
+            false}
```

Comment:
some of this formatting looks funky

Reply:
Linters are preventing me from keeping this on one line 😭